games-action/endless_sky/endless_sky-0.9.10.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="A 2D space trading and combat game"
+DESCRIPTION="Endless Sky is a 2D space trading and combat game \
+similar to the classic Escape Velocity series. Explore other star systems. \
+Earn money by trading, carrying passengers, or completing missions. \
+Use your earnings to buy a better ship or to upgrade the weapons and engines \
+on your current one. Blow up pirates. Take sides in a civil war. \
+Or leave human space behind and hope to find friendly aliens whose culture \
+is more civilized than your own."
+HOMEPAGE="http://endless-sky.github.io/"
+COPYRIGHT="Michael Zahniser"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/endless-sky/endless-sky/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="58f2e405e171be8426004216df4df7bb3a2792a507a5d01f77a0f02346332f6f"
+SOURCE_DIR="endless-sky-$portVersion"
+PATCHES="endless_sky-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 !x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	endless_sky$secondaryArchSuffix = $portVersion
+	app:EndlessSky$secondaryArchSuffix = $portVersion
+"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libglew$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libmad$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
```

Comment:
Suggestion:
```
	lib:libGL$secondaryArchSuffix
	lib:libglew$secondaryArchSuffix
	lib:libjpeg$secondaryArchSuffix
	lib:libmad$secondaryArchSuffix
	lib:libopenal$secondaryArchSuffix
	lib:libpng16$secondaryArchSuffix
	lib:libSDL2_2.0$secondaryArchSuffix
```

Reply:
Same for devel:*

Reply:
@Begasus that still doesn't work on Nightly.

@scottmc here's the game crashing on startup in a Nightly 32bit:https://snipboard.io/4O2RqZ.jpg and what it outputs in the debugger:  https://snipboard.io/NjYOwJ.jpg
and here is it running on r1Beta1 32bit: https://snipboard.io/ZfeSma.jpg

Reply:
It crashes also on R1Beta1 64bits